src/hec/firo/EnsembleDatabase.java

```diff
@@ -10,10 +9,10 @@
 /**
  *  Read/Write Ensembles to a ODBC database
  */
-public class EnsembleDatabase {
+public class EnsembleDatabase implements AutoCloseable {
 
     static String DateTimeFormat = "yyyy-MM-dd HH:mm:ss";
-    private static String TableName = "timeseries_blob";
+    private static String TableName = "timeseries_ensemble";
```

Comment:
that is clearer. is it a single time series_ensemble or is it many time series_ensembles?

Reply:
The table holds many time series.  The singular 'ensemble' versus 'ensembles' is a preference, when thinking ahead to an API.

for example:   timeseries_ensemble[12]